src/webui/www/private/scripts/client.js

```diff
@@ -744,21 +744,21 @@ window.addEvent('load', function() {
         }
 
         switch (serverState.connection_status) {
```

Comment:
Indentation is broken inside this `switch` statement.

Reply:
Seems the formatting tool is confused by placing `break;` outside the curly brackets. I removed the curly brackets.

Reply:
I never fully trusted all these formatting tools...
But in any case, I'm relaxed about the pure WebUI changes. Just once again reminded you to be attentive to the results of auto-formatting.